ambry-mysql/src/integration-test/java/com/github/ambry/accountstats/AccountStatsMySqlStoreIntegrationTest.java

```diff
@@ -60,10 +60,13 @@
 public class AccountStatsMySqlStoreIntegrationTest {
   private static final String clusterName1 = "Ambry-test";
   private static final String clusterName2 = "Ambry-random";
+  // hostname1 and hostname2 are the same, but with different port numbers
   private static final String hostname1 = "ambry1.test.github.com";
-  private static final String hostname2 = "ambry2.test.github.com";
+  private static final String hostname2 = "ambry1.test.github.com";
```

Comment:
you can remove `hostname2` then

Reply:
I like to keep it so that we know the difference between these two instances.